### ISSUE TITLE
Screen reader text in block library shouldn't be visible on focus

### ISSUE THREAD

**Issue by walbo** [WordPress/gutenberg#33724]
## Description
#30141 added `screen-reader-text` in the common block library CSS.

`:focus` on `screen-reader-text` has some unintended effects on some other focusable elements in core, and users' websites could have something similar on the front end. See core ticket https://core.trac.wordpress.org/ticket/53803

The focus styles are intended mainly for [skip links](https://make.wordpress.org/accessibility/2015/02/09/hiding-text-for-screen-readers-with-wordpress-core/), not just anything with the class. 

cc @sabernhardt 

## WordPress information
- WordPress version: 5.8
- Gutenberg version: Not installed
- Are all plugins except Gutenberg deactivated? Yes
- Are you using a default theme (e.g. Twenty Twenty-One)? Yes

## Device information
- Device: Desktop
- Operating system: Mac OS Big Sur
- Browser: Chrome 92

**Comment by youknowriad** [comment]
Going to close this issue as the related PR has also been closed. Let me know if you think this is still an issue.

**Comment by sabernhardt** [comment]
I think the `common.scss` should have been edited more than four years ago, but changing it now could pose a greater risk. There probably are some sites that rely on the `.screen-reader-text:focus` styles to show elements (without the `.skip-link` class).
